code/modules/antagonists/_common/antag_spawner.dm

```diff
@@ -10,6 +10,63 @@
 /obj/item/antag_spawner/proc/equip_antag(mob/target)
 	return
 
+//sleeper agents for nuke ops
+/obj/item/antag_spawner/sleeper_activator
+	name = "sleeper agent activation remote"
+	desc = "A remote used to activate a deep cover sleeper agent aboard the Cyberiad, which will aid us from the inside."
+	icon = 'icons/obj/device.dmi'	//same as borg tele
+	icon_state = "locator"
+	var/checking = FALSE
+	var/desired_objective
+
+/obj/item/antag_spawner/sleeper_activator/attack_self(mob/user)
+	if(used)
+		to_chat(user, "<span class='warning'>[src] has already been used.</span>")
+		return
+	if(!(user.mind in ticker.mode.syndicates))
+		to_chat(user, "<span class='danger'>AUTHENTICATION FAILURE. ACCESS DENIED.</span>")
+		return
+	if(checking)
+		to_chat(user, "<span class='warning'>[src] is already checking for sleeper agents to activate!</span>")
+		return
+	desired_objective = stripped_input(user, "What objective do you wish to give the sleeper agent?", "Objective")
+	checking = TRUE
```

Comment:
I think this `checking` bit needs to go above the `stripped_input`, or you can click it lots of times to get lots of dialogue boxes and therefore lots of sleeper agents

Reply:
Nice catch, I didn't notice that one.